repositories.bzl

```diff
@@ -119,6 +122,13 @@ def com_google_auth_google_auth_library_credentials():
         sha1 = "8e2b181feff6005c9cbc6f5c1c1e2d3ec9138d46",
     )
 
+def com_google_auth_google_auth_library_oauth2_http():
+    native.maven_jar(
+        name = "com_google_auth_google_auth_library_oauth2_http",
+        artifact = "com.google.auth:google-auth-library-oauth2-http:0.9.0",
```

Comment:
Is there a better library to depends on? DefaultChannel needs to call GoogleCredentials.getApplicationDefault().

Reply:
To my knowledge... no.

Reply:
Shall we add a comment on ALTS that alts artifact will depend on all these?

Reply:
No, it would just be a dependency in the BUILD.bzl file. Note that if we were doing this "right" we would have a third_party/ directory that would pull in the maven_jar and include all the proper dependencies.